service/crt/service.go

```diff
@@ -28,9 +27,8 @@ const (
 	CertificateWatchAPIEndpoint string = "/apis/giantswarm.io/v1/watch/certificates"
 
 	// resyncPeriod is the period for re-synchronizing the list of objects in k8s
-	// watcher. 0 means that re-sync will be delayed as long as possible, until
-	// the watch will be closed or timed out.
-	resyncPeriod time.Duration = 0
+	// watcher. Set to 1 minute to make the watch more robust.
+	resyncPeriod time.Duration = time.Minute * 1
```

Comment:
AFAIK the resync is a corrective action so the watch stays healthy since its a long running process. So I'd say 1 second is too often. The lowest value I've seen is 15 seconds in the Rook operator.

I agree on needing to experiment on this. I could use a flag to make that easier?

This is the best link I can find on what the resync period is for. The current value of 0 which delays the resync for as long as possible feels wrong to me.

https://github.com/kubernetes/kubernetes/issues/26406